core/src/test/java/bio/terra/pearl/core/service/survey/SurveyServiceTests.java

```diff
@@ -422,12 +422,12 @@ public void testCreateDeleteWithLanguageTexts(TestInfo info) {
                 .eligibilityRule("")
                 .build();
         survey = surveyService.create(survey);
-        List<LanguageText> languageTexts = languageTextDao.findByPortalIdOrNullPortalId(portal.getId(), "es");
+        List<LanguageText> languageTexts = languageTextDao.findByPortalId(portal.getId(), "es");
         assertThat(languageTexts, hasSize(1));
         assertThat(languageTexts.get(0).getKeyName(), equalTo(survey.getStableId() + ":1"));
         surveyService.delete(survey.getId(), CascadeProperty.EMPTY_SET);
 
-        assertThat(languageTextDao.findByPortalIdOrNullPortalId(portal.getId(), "es"), hasSize(0));
+        assertThat(languageTextDao.findByPortalId(portal.getId(), "es"), hasSize(0));
     }
```

Comment:
drive by flaky test fix - it looks like some other test is adding a bunch of stuff to the `null` portal and not cleaning it up, so depending on test execution it can fail